fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/recordrepair/RecordValidateOnlyTest.java

```diff
@@ -248,7 +251,9 @@ void testValidateRecordsCorruptRecord(FormatVersion formatVersion, RecordRepair.
         ValidationTestUtils.assertInvalidResults(
                 repairResults.getInvalidResults(),
                 1,
-                result -> result.equals(RecordRepairResult.invalid(primaryKey, RecordRepairResult.CODE_DESERIALIZE_ERROR, "any")));
+                result -> !result.isValid() &&
+                        result.getErrorCode().equals(RecordRepairResult.CODE_DESERIALIZE_ERROR));
+        Assertions.assertThat((Object)primaryKey).isEqualTo(repairResults.getInvalidResults().get(0).getPrimaryKey());
```

Comment:
```suggestion
        Assertions.assertThat(repairResults.getInvalidResults().get(0).getPrimaryKey()).isEqualTo((Object)primaryKey);
```

(although, now having pointed out a few of these, you _may_ want to have a method that asserts about the only primary key of the invalid results).

Reply:
Done. I think being explicit about the one primary key is OK for now. The tests are much more readable and in better shape than they used to be.